appletun/__main__.py

```diff
@@ -0,0 +1,242 @@
+                                               'HTTPProxy': http_proxy_addr,
+                                               'HTTPSEnable': True,
+                                               'HTTPSPort': http_proxy_port,
+                                               'HTTPSProxy': http_proxy_addr},
+                                   'UserDefinedName': vpn_name,
+                                   'VPNType': 'IKEv2'}],
+               }
+    if http_proxy_cert is not None:
+        profile['PayloadContent'].append({'PayloadIdentifier': 'com.apple.security.root.86FD9307-9CAC-4401-ABD4-FE0F70521A62',
+                                          'PayloadType': 'com.apple.security.root',
+                                          'PayloadUUID': '86FD9307-9CAC-4401-ABD4-FE0F70521A62',
+                                          'PayloadVersion': 1,
+                                          'PayloadContent': http_proxy_cert})
+    return profile
+
+def generate_and_install_profile(service: MobileConfigService,
+                                 server_address: Optional[str] = None,
+                                 vpn_name: str = "AppleTun",
+                                 psk: Optional[str] = None,
+                                 http_proxy_port: int = 9090,
+                                 http_proxy_addr: Optional[str] = None,
+                                 http_proxy_cert: Optional[bytes] = None,
+                                 write_config: bool = True):
+    """ Generate and install a VPN profile """
+    config_dir = get_vpn_config_dir()
+    if psk is None:
+        vpn_secrets = get_vpn_secret(config_dir, vpn_name)
+        if len(vpn_secrets) == 0:
+            characters = string.ascii_letters + string.digits + string.punctuation
+            psk = ''.join(secrets.choice(characters) for _ in range(12))
+        else:
+            psk = vpn_secrets[0]
+    server_address = server_address if server_address is not None else get_machine_ips(ipv4=True, ipv6=False)[0]
+    http_proxy_addr = http_proxy_addr if http_proxy_addr is not None else server_address
+
+    profile = generate_profile(server_address, vpn_name, psk, http_proxy_port, http_proxy_addr, http_proxy_cert)
+    service.install_profile(plistlib.dumps(profile))
+
+    if write_config:
+        write_new_vpn_config(config_dir, vpn_name)
+        write_vpn_secret(config_dir, vpn_name, psk)
+
+@click.group()
+def cli():
+    pass
+
+@cli.command(cls=Command)
+def start(service_provider:   LockdownServiceProvider):
+    """ Start AppleTun VPN """
+    service = MobileConfigService(lockdown=service_provider)
+    profiles = service.get_profile_list()
+    if PROFILE_UUID in profiles["ProfileManifest"].keys():
+        description = profiles["ProfileManifest"][PROFILE_UUID]["Description"]
+        print(f"installed profile: {description}")
+    else:
+        print("Warning: generated AppleTun profile not found")
+
+    local["ipsec"]("restart")
```

Comment:
```suggestion
    ipsec = local["sudo"]["ipsec"]
    ipsec("restart")
```

Make `ipsec` into a global (it will act as just anohter function within the module)